lms/djangoapps/courseware/views.py

```diff
@@ -866,6 +867,9 @@ def course_about(request, course_id):
         # get prerequisite courses display names
         pre_requisite_courses = get_prerequisite_courses_display(course)
 
+        # Overview
+        overview = CourseOverview.get_from_id(course.id)
```

Comment:
By the way, have you looked into replacing `course` with `overview` in this method, eliminating the need to call `get_course_with_access`?  It's out of the scope of this PR, but just wondering if you checked.